build/index.js

```diff
@@ -110,6 +112,24 @@ examples.forEach(example => {
   console.log('saved to file examples/' + fileName)
 })
 
+// Write revelant playground fixtures to a Jekyll collection
+fs.emptyDir(fixturesCollection, err => {
+  axe.getRules().forEach(rule => { // only fixtures with a corresponding axe rule
+    const filename = path.join(root, './playground', 'fixtures', rule.ruleId) + '.html'
+    fs.readFile(filename, 'utf8', (err, data) => {
+      if (!err && data && data.length) {
+        frontLoad(
+          path.join(root, fixturesCollection, rule.ruleId) + '.html',
```

Comment:
This allows us to keep raw HTML test fixtures in our source without worrying about whether they each correspond with an existing aXe rule. This build step will create the collection for Jekyll comprised of only those fixtures which have a corresponding aXe rule. It keeps us in sync with aXe while allowing us to curate and modify an arbitrary list of test fixtures.